grip/server.py

```diff
@@ -384,9 +392,8 @@ def _cache_contents(style_urls, asset_pattern, asset_pattern_sub, cache_path):
 
     for asset_url in asset_urls:
         filename = _cache_filename(asset_url, cache_path)
-        contents = requests.get(asset_url, verify=False).text
-        # Write file and show message
-        _write_file(filename, contents)
+        # Retrieve file and show message
+        urlretrieve(asset_url, filename)
```

Comment:
Is this more robust than using requests and writing to a file?

Requests handles things like following redirects for you.